EpsilonCCS/Inc/Data/BatteryData.h

```diff
@@ -23,7 +25,7 @@ enum BatteryPrechargeState
 
 struct BatteryData
```

Comment:
So the goal was to have the CanParsers match the incoming CAN messages, and everything beyond that match the serial protocol. I think its' too much overhead now to change the serial protocol and I don't think there's much benefit anyways.